app/src/main/java/io/intrepid/contest/base/BaseViewPagerFragment.java

```diff
@@ -0,0 +1,13 @@
+package io.intrepid.contest.base;
+
+public abstract class BaseViewPagerFragment<T extends BaseContract.Presenter> extends BaseFragment<T> implements TextValidatableView {
```

Comment:
It will be cleaner to remove the BaseviewPagerFragment and instead just implement the interface wherever we need it. The functions below wont ever actually be used in this format so its creating unnecessary code.